### PR TITLE
[BM] split chat direct messages

### PR DESCRIPTION
**Pull request by ermiaSparkle**
Resolve issue https://github.com/sparkletown/internal-sparkle-issues/issues/1006

Wait for PR:
- https://github.com/sparkletown/sparkle/pull/2007

### CODE REVIEW

File: src/components/organisms/ChatSidebar/ChatSidebar.scss [hunk 1]
```diff
@@ -89,6 +89,17 @@ $header-height: 45px;
...
+
+    &--new-message {
+      background-color: #f44336;
+      height: 12px;
+      width: 12px;
+      border: 2px solid rgba($saturated-black, 0.9);
+      position: absolute;
+      border-radius: $border-radius--max;
+      right: 8px;
+      bottom: 10px;
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.scss:101]: Constants for colors and dimensions.

Recommend `opaque-black()` function for the border color if it is close enough to `$saturated-black`

File: src/components/organisms/ChatSidebar/ChatSidebar.scss [hunk 2]
```diff
@@ -89,6 +89,17 @@ $header-height: 45px;
     &__second-icon {
       margin-left: 5px;
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.scss:90]: How to deal with the redundant class and spacing literal:
https://github.com/sparkletown/sparkle/pull/2007/files#diff-977ce0dba740bcc42e948ea6052d729b246d573391d10f3a95fb9c1a7448df98L89-R97

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 3]
```diff
@@ -84,6 +87,26 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
             </>
           )}
         </button>
+        {!isExpanded && (
+          <button
+            aria-label={isExpanded ? "Hide chat" : "Show chat"}
+            className="chat-sidebar__controller"
+            onClick={togglePrivateChatSidebar}
+            style={{ marginTop: 55 }}
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:95]: Style property values should be part of the SCSS classes i.e. no inline styles unless there's no way around it.

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 4]
```diff
@@ -84,6 +87,26 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
...
+          >
+            <>
+              <FontAwesomeIcon icon={faChevronLeft} size="sm" />
+              <FontAwesomeIcon
+                className="chat-sidebar__controller__second-icon"
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:100]: Class name serving only for `:nth-child(2)` pseudo selector

**Review by goran-peoski-work — COMMENTED** [review]
Few notes for improvements

**Comment by ermiaSparkle** [comment]
Wait for PR:
- https://github.com/sparkletown/sparkle/pull/2007

**Comment by ermiaSparkle** [comment]
Branch got dirty so this one will be deleted.

**Comment by whoaitsaimz** [comment]
Replaced by 
- https://github.com/sparkletown/sparkle/pull/2052

**Pull request closed without merging**